src/frontend/src/icp/utils/ic-send.utils.ts

```diff
@@ -30,14 +31,19 @@ export const isBtcAddress = (address: BtcAddress | undefined): boolean => {
 export const invalidBtcAddress = (address: BtcAddress | undefined): boolean =>
 	!isBtcAddress(address);
 
+const isTokenCkLedger = (
```

Comment:
- `isTokenLedger`?

- Can you use an object as parameter?

Reply:
I prefer

```
export const isTokenCkBtcLedger = (token: Partial<IcToken>): boolean =>
	isTokenLedger({token, ledgerIds: CKBTC_LEDGER_CANISTER_IDS});
```

Reply:
Btw. if you are up to setup an eslint rules when you got a bit of time, that would be super.

Reply:
About this one? what would you put as rule?

Reply:
> Always pass an object unless one single parameter?

I would say so, what do you think?

Reply:
I like it! Even because I didn't know what was the convention eheh

Reply:
In the meanwhile, corrected this one, have a look please

Reply:
Done with PR:

- #2416